src/libertem/common/buffers.py

```diff
@@ -561,6 +570,28 @@ def __repr__(self):
             self._kind, self._dtype, self._extra_shape
         )
 
+    def update_data(self, data: nt.ArrayLike, force: bool = False) -> None:
```

Comment:
```suggestion
    def update_data(self, data: "nt.ArrayLike", force: bool = False) -> None:
```

Using a string here delays the evaluation to type checking time.

Reply:
I've implemented this change, thanks!